dicoogle/src/main/resources/webapp/js/components/indexer/IndexStatusView.js

```diff
@@ -28,20 +45,22 @@ const IndexStatusView = React.createClass({
          //Stop refresh interval
          clearInterval(refreshIntervalId);
 
+         this.unsubscribeProviders;
+         this.unsubscribeIndexStatus;
+         this.unsubscribePlugin;
```

Comment:
Oops! Parentheses are missing. It should be `this.unsubscribeProviders();` and the same to the other two lines.

Reply:
Oops x) Thank you!